packages/babel-plugin-proposal-decorators/test/fixtures/2023-11-misc/decorator-evaluation-super-property/exec.js

```diff
@@ -0,0 +1,148 @@
+function noop() {}
+
+{
+  let receivedName;
+  class B extends class A {} {
+    static m() {
+      class C {
+        @(receivedName = super.name, noop) p;
+      }
+    }
+  }
+
+  B.m();
+  expect(receivedName).toBe("A");
+}
+
+{
+  let receivedName;
+  class B extends class A {} {
+    static m() {
+      class C {
+        @(receivedName = super.name, noop) #p;
+      }
+    }
+  }
+
+  B.m();
+  expect(receivedName).toBe("A");
+}
+
+{
+  let receivedName;
+  class B extends class A {} {
+    static m() {
+      class C {
+        @(receivedName = super.name, noop) accessor [(noop(), "p")];
+      }
+    }
+  }
+
+  B.m();
+  expect(receivedName).toBe("A");
+}
+
+{
+  let receivedName;
+  class B extends class A {} {
+    static m() {
+      @noop
+      class C {
+        @(receivedName = super.name, noop) p;
```

Comment:
Can you add a test with a decorator that references both `super` and a private field in the same decorator?

Reply:
Does this test look good?

https://github.com/babel/babel/blob/0b8f0bc3acc8ff3f44f0fa6931d6bad3dec9c0a3/packages/babel-plugin-proposal-decorators/test/fixtures/2023-11-misc--to-es2015/decorator-evaluation-yield-private-super-property/exec.js#L12-L29

Reply:
This is ok, I somehow missed it.